git-some.sh

```diff
@@ -18,6 +18,12 @@ if grep --quiet --invert-match '^[[:digit:]]*$' <<< $commit_count; then
   exit 1
 fi
 
+COUNT=$(/bin/find . -maxdepth 1 -name "file-??.txt" | wc --lines)
+if [[ $(($COUNT + $commit_count)) -gt 100 ]]; then
```

Comment:
Extract upper limit of`100` files into a variable and also use that variable with the random number generator in https://github.com/GROSSWEBER/git-some/blob/master/git-some.sh#L24